common/src/main/scala/org/neo4j/spark/service/MappingService.scala

```diff
@@ -70,8 +73,8 @@ class Neo4jWriteMappingStrategy(private val options: Neo4jOptions)
     if (nodeMetadata.nodeKeys.contains(key)) {
       nodeMap.get(KEYS).put(nodeMetadata.nodeKeys.getOrElse(key, key), value)
     }
-    if (nodeMetadata.nodeProps.contains(key)) {
-      nodeMap.get(PROPERTIES).put(nodeMetadata.nodeProps.getOrElse(key, key), value)
+    if (nodeMetadata.properties.contains(key)) {
```

Comment:
missing `else`?

Reply:
No, it's not if/else alternative we need to check for both `nodeKeys` and `properties`, I guess that we can change the method in this way:

```scala
    if (nodeMetadata.nodeKeys.contains(key)) {
      nodeMap.get(KEYS).put(nodeMetadata.nodeKeys.get(key), value)
    }
    if (nodeMetadata.properties.contains(key)) {
      nodeMap.get(PROPERTIES).put(nodeMetadata.properties.get(key), value)
    }
```